Python/Product/Core/Core.csproj

```diff
@@ -132,11 +131,6 @@
     <None Include="source.extension.vsixmanifest" />
     <Content Include="License_en-US.rtf" />
     <Content Include="$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll" />
-    <Content Include="$(PackagesPath)\Microsoft.VisualStudio.LanguageServer.Protocol\lib\netstandard2.0\Microsoft.VisualStudio.LanguageServer.Protocol.dll" />
```

Comment:
Protocol.dll removed on purpose?

Reply:
Sorry should have mentioned it too. We don't need to ship it anymore either.